workspace-hack/Cargo.toml

```diff
@@ -40,11 +40,13 @@ num-traits = { version = "0.2.15", features = ["i128", "libm", "std"] }
 pbkdf2 = { version = "0.9.0", default-features = false, features = ["hmac", "sha2"] }
 ppv-lite86 = { version = "0.2.16", default-features = false, features = ["simd", "std"] }
 rand = { version = "0.8.5", features = ["alloc", "getrandom", "libc", "rand_chacha", "std", "std_rng"] }
-rand_chacha = { version = "0.3.1", default-features = false, features = ["std"] }
-rand_core = { version = "0.6.3", default-features = false, features = ["alloc", "getrandom", "std"] }
+rand_chacha = { version = "0.3.1", default-features = false, features = ["serde", "serde1", "std"] }
+rand_core-d8f496e17d97b5cb = { package = "rand_core", version = "0.5.1", default-features = false, features = ["alloc", "getrandom", "std"] }
+rand_core-3b31131e45eafb45 = { package = "rand_core", version = "0.6.3", default-features = false, features = ["alloc", "getrandom", "std"] }
```

Comment:
Any idea why we have two versions of `rand_core` at the same time?

Reply:
Just some internal part of the dependency tree, not sure why.